python/GafferImageUI/ImageStatsUI.py

```diff
@@ -39,18 +39,6 @@
 import GafferImage
 import GafferImageUI
 
-## A function suitable as the postCreator in a NodeMenu.append() call. It
-# sets the region of interest for the node to cover the entire format.
-def postCreate( node, menu ) :
```

Comment:
Technically, removing this is an API break. I find it hard to imagine someone using this function, but it _is_ part of the public API.